docs/wetgeving.md

```diff
@@ -0,0 +1,46 @@
+# Open Source in relatie tot wetgeving
+
+Open Source is op zichzelf geen doel van een rechtmatige overheid. Open Source kan wel een middel zijn
+om als overheid rechtmatig te handelen. De direct betrok wetgeving zijn de Wet open overheid (Woo),
+Wet hergebruik overheidsinformatie (Who), Auteurswet en de Wet markt en overheid (W M&O).
+
+Over het auteursrecht is al voldoende beschreven bij licenties elders hier in de kennisbank.
```

Comment:
```suggestion
Over het auteursrecht is al voldoende geschreven op [de type licenties pagina](https://ospo-nl.github.io/kennisbank/Best%20practices/Type%20licenties/)
```

Reply:
Een relatieve link is voldoende 😉 

> Over het auteursrecht is al voldoende beschreven op de [licenties pagina](../Best%20practices/Type%20licenties/).

Reply:
Goed idee om er een relatieve link van te maken.